assemblyline/templates/ui.yaml

```diff
@@ -58,9 +59,11 @@ spec:
           env:
           {{ include "assemblyline.coreEnv" . | indent 12 }}
             - name: WORKERS
-              value: '6'
-            - name: WORKER_CONNECTIONS
-              value: '2048'
+              value: '2'
```

Comment:
We've determined with a lot of throughput testing on our end that the optimal value for the number of workers was 6 with the max connections to be 2048. I'd rather this not being changed here or make it configurable if you really want to change those values.